<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleLogin</class>
 <widget class="QMainWindow" name="SimpleLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleLogin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="signinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="title">
     <string>SignIn</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userNameForm_l"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>password </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordForm_l">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="login">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="signupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="title">
     <string>Signup</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userNameForm"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>password </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordForm">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="signup">
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     background-color: gray; border: 1px solid gray;
	color: #ffffff;}</string>
    </property>
    <property name="text">
     <string>Simple Login Service</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>signinBox</zorder>
   <zorder>Title</zorder>
   <zorder>signupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
